config/default_records/request_types/018_limber_lcm_triomics_request_types.wip.yml

```diff
@@ -0,0 +1,12 @@
+# Request types for LCM Triomics DNA WGS and EMSeq
+---
+limber_lcm_triomics:
+  name: LCM Triomics
+  asset_type: Well
+  order: 1
+  request_class_name: CustomerRequest
```

Comment:
Probably needs to be a library request, and include a library type. There's an existing library type called 'emSEQ', used in conjunction with the existing Limber PCR Bespoke pipeline. Not sure if it's appropriate to use that one, or if we should make a new one. Probably needs discussion with NPG as they use the library type to decide which analysis pathway to send it down.